### PR TITLE
added test case for `internal/net/http/routing`

### PR DESCRIPTION
**Pull request by hlts2**
### Description:

I added test case for `internal/net/http/routing`
I have confirmed that go test is passed.

### Environment:

- Golang Version: 1.13.5
- Docker Version: 19.03.5
- Kubernetes Version: 1.16.3
- NGT Version: 1.8.4

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [x] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [x] I have added tests and benchmarks to cover my changes.
- [x] I have ensure all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: internal/net/http/routing/router_test.go [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+			cnt := 0
+			h := func(w http.ResponseWriter, req *http.Request) (code int, err error) {
+				cnt++
+				w.WriteHeader(http.StatusBadRequest)
+				return http.StatusOK, errors.New("faild")
+			}
+
+			return test{
+				name: "internal call error",
```
Comment (vankichi) [inline internal/net/http/routing/router_test.go]: - Could you rename test name following guidelines ?

File: internal/net/http/routing/router_test.go [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+				},
+			}
+		}(),
+
+		func() test {
+			w := new(httptest.ResponseRecorder)
+			r := httptest.NewRequest(http.MethodGet, "/", new(bytes.Buffer))
+
+			return test{
+				name: "invalid request method",
```
Comment (vankichi) [inline internal/net/http/routing/router_test.go]: - Could you rename test name following guidelines ?

Comment (hlts2) [reply internal/net/http/routing/router_test.go]: Thank you for your review:bow:
I Fixed :bow:

File: internal/net/http/routing/router_test.go (hunk 1 above)
Comment (hlts2) [reply internal/net/http/routing/router_test.go]: Thank you for your review🙇
I Fixed 🙇

**Pull request merged**